QUBEKit/forcefield/force_groups.py

```diff
@@ -23,6 +23,16 @@ class BaseForceGroup(BaseModel):
     type: Literal["base"] = "base"
     parameters: Optional[Dict[Tuple[int, ...], Type[BaseParameter]]] = None
 
+    def __iter__(self) -> Generator:
+        for parameter in self.parameters.values():
+            yield parameter
+
+    def __getitem__(self, item):
```

Comment:
I think we want to raise a MissingParameterError here if the parameter is not found